src/components/panes/ImportLogPane.jsx

```diff
@@ -0,0 +1,135 @@
+        state.importer.importLogList,
+        props.paneData.params[0])
+}))
+@injectIntl
+export default class ImportLogPane extends PaneBase {
+    getPaneTitle(data) {
+        return this.props.intl.formatMessage({ id: 'panes.importLog.title' });
+    }
+
+    getRenderData() {
+        return this.props.importLog;
+    }
+
+    renderPaneContent(data) {
+        const log = data.data;
+        if (!log) {
+            const infoListData = [
+                { name: 'error', msgId: 'panes.importLog.error.noImportLog' }
+            ];
+
+            return (<div key="info" className="ImportLogPane-info">
+                <InfoList
+                    data={ infoListData }
+                />
+            </div>);
+        } else {
+            let infoListData = [
+                {
+                    name: 'status',
+                    msgId: 'lists.importLogList.item.status.' + log.status
+                },
+                {
+                    name: 'accepted',
+                    msgId: 'panes.importLog.accepted',
+                    msgDatetimes: { accepted: log.accepted },
+                }, 
+            ];
+
+            if(log.status == 'completed') {
+                let completedData = [
+                    {
+                        name: 'completed',
+                        msgId: 'panes.importLog.completed',
+                        msgDatetimes: { completed: log.completed }
+                    },
+                    {
+                        name: 'imported',
+                        msgId: 'panes.importLog.report.imported',
+                        msgNumbers: { imported: [log.report.imported] },
+                    },
+                    {
+                        name: 'created',
+                        msgId: 'panes.importLog.report.created',
+                        msgNumbers: { created: [log.report.created] },
+                    },
+                    {
+                        name: 'updated',
+                        msgId: 'panes.importLog.report.updated',
+                        msgNumbers: { updated: [log.report.updated] },
+                    },
+                    {
+                        name: 'tagged',
+                        msgId: 'panes.importLog.report.tagged',
+                        msgNumbers: { tagged: [log.report.tagged] },
+                    },
+                    {
+                        name: 'taggings',
+                        msgId: 'panes.importLog.report.taggings',
+                        msgNumbers: { taggings: [log.report.taggings] },
+                    },
+                ];
+                infoListData = infoListData.concat(completedData);
+            } else if(log.status == 'error') {
+                infoListData.push(
+                    {
+                        name: 'error',
+                        msgId: 'panes.importLog.error',
+                        value: log.error
+                    },
+                );
```

Comment:
Is it really sensible to print out the error as stored in the database (line 94)? For one of my failed imports, this is what it looked like:

![image](https://user-images.githubusercontent.com/550212/78457547-4c4fba00-76ab-11ea-8648-314810578666.png)